assets/styles/main.scss

```diff
@@ -32,6 +32,10 @@
 @import "generic/form";
 @import "generic/button";
 
+// Vendors
+// ==========================================================================
+@import "vendors/locomotive-scroll";
```

Comment:
I don't think it's necessary to create a new file in the styles vendors. I would just import the vendor here!